solver/pUf/pEqn.H

```diff
@@ -254,11 +254,11 @@ License
             // Set pressure reference
             pEqn.setReference(pRefCell, getRefCellValue(p_rbgh, pRefCell));
 
-#if (defined(OFVERSION) && (OFVERSION >= 700))
+//#if (OFVERSION > 700)
             pEqn.solve();
-#else
-            pEqn.solve(mesh.solver(p_rbgh.select(pimple.finalInnerIter())));
-#endif
+//#else
+//            pEqn.solve(mesh.solver(p_rbgh.select(pimple.finalInnerIter())));
+//#endif
```

Comment:
again.... why are you deleting those lines necessary to support multiple versions of openfoam?